ActivityTracker/background.js

```diff
@@ -204,34 +249,43 @@ class StatisticsHandler {
     }
 
     reset() {
-		this.hostnameToTimeData = {};
+        this.hostnameToTimeData = {};
         this.lastHostname = '';
     }
 }
 
-let gStatisticsHandler = new StatisticsHandler();
+window.statisticsHandler = new StatisticsHandler();
 
 function updateStatisticsFromTab(tab) {
     if (!tab.url || tab.url.length === 0) {
         return;
     }
     const { url, id } = tab;
     const hostname = getUrlHostname(url);
-    console.log('onUpdated url: ' + url);
-    console.log('onUpdated hostname: ' + hostname);
-    gStatisticsHandler.updateHostTimeData(hostname, id);
+    if(!hostname.length) {
+        console.warn("Tab with empty host: " + url + " skip update");
```

Comment:
here you're saying `skip update`, but the function keeps on executing. `window.statisticsHandler.updateHostTimeData(hostname, id);` will be eventually called.
Is this expected?

Reply:
I'll add return